control-plane/subcommand/server-acl-init/create_or_update.go

```diff
@@ -152,40 +153,30 @@ func (c *Command) updateOrCreateBindingRule(client *api.Client, authMethodName s
 		return err
 	}
 
-	// If the binding rule already exists, update it
-	// This updates the binding rule any time the acl bootstrapping
-	// command is rerun, which is a bit of extra overhead, but is
-	// necessary to pick up any potential config changes.
-	if len(existingRules) > 0 {
-		// Find the policy that matches our name and description
-		// and that's the ID we need
-		for _, existingRule := range existingRules {
-			if existingRule.BindName == abr.BindName && existingRule.Description == abr.Description {
-				abr.ID = existingRule.ID
-			}
-		}
-
-		// This will only happen if there are existing policies
-		// for this auth method, but none that match the binding
-		// rule set up here in the bootstrap method.
-		if abr.ID == "" {
-			return errors.New("unable to find a matching ACL binding rule to update")
+	// Find the policy that matches our name and description
+	// and that's the ID we need
+	for _, existingRule := range existingRules {
+		if existingRule.BindName == abr.BindName && existingRule.Description == abr.Description {
+			abr.ID = existingRule.ID
 		}
+	}
 
-		err = c.untilSucceeds(fmt.Sprintf("updating acl binding rule for %s", authMethodName),
-			func() error {
-				_, _, err := client.ACL().BindingRuleUpdate(abr, nil)
-				return err
-			})
-	} else {
-		// Otherwise create the binding rule
-		err = c.untilSucceeds(fmt.Sprintf("creating acl binding rule for %s", authMethodName),
+	// This will happen in these circumstances:
+	// 1) there are no existing policies
+	// 2) there are existing policies for this auth method, but none that match
+	//    the binding rule set up here in the bootstrap method.
+	if abr.ID == "" {
+		return c.untilSucceeds(fmt.Sprintf("creating acl binding rule for %s", authMethodName),
```

Comment:
I thought we've decided that we no longer need this logic because component binding rule is created by a different method?

Reply:
yes, thank you.  this is a remnant. will remove.